### ISSUE TITLE
Wrong time is used in pidor detection functionality

### ISSUE THREAD

**Issue by vasua** [Assada/kiev_leprobot#6]
In UserChatRepository code for time boundaries determination looks like this: 
```    
let end = new Date(new Date() + 2 * 3600 * 1000).toISOString().slice(0, 19).replace('T', ' ');
let start = new Date(new Date() - 24 * 60 * 60 * 1000 + 2 * 3600 * 1000).toISOString().slice(0, 19).replace('T', ' ');
```
This code will provide following result
```
console.log({start,end});
{start: "2021-02-10 00:41:50", end: "2021-02-10 22:41:50"}
```

The suggestion would be to use `new Date().getTime()` instead of `new Date()`. That seems to work pretty fine:
```
let end = new Date(new Date().getTime() + 2 * 3600 * 1000).toISOString().slice(0, 19).replace('T', ' ');
let start = new Date(new Date().getTime() - 24 * 60 * 60 * 1000 + 2 * 3600 * 1000).toISOString().slice(0, 19).replace('T', ' ');
console.log({start,end});
{start: "2021-02-10 00:42:04", end: "2021-02-11 00:42:04"}
```

**Comment by assada** [comment]
Resolved in f8e934a
Thx!

**Comment by assada** [comment]
Also - Released at prod
